PRIVACY.md

```diff
@@ -0,0 +1,93 @@
+As part of offering and providing customizable and personalized services, Konflux uses cookies to store and sometimes track information about you. A cookie is a small amount of data that is sent to your browser from a Web server and stored on your computer's hard drive.
+Generally, we use cookies to:
+
+2. Estimate audience size. Each browser accessing Konflux is given a unique cookie that is used to determine the extent of repeat usage and usage by a registered user versus by an unregistered user.  
+4. Measure certain traffic patterns, which areas of Konflux's network of websites you have visited, and your visiting patterns in the aggregate. We use this research to understand how our user's habits are similar or different from one another so that we can make each new experience on Konflux website a better one. We may use this information to better personalize the content, banners, and promotions you and other users will see on our sites.
+
+If you do not want your personal information to be stored by cookies, you can configure your browser so that it always rejects these cookies or asks you each time if you accept them or not. However, you must understand that the use of cookies may be necessary to provide certain services, and choosing to reject cookies will reduce the performance and functionality of the site. Your browser documentation includes instructions explaining how to enable, disable or delete cookies at the browser level (usually located in the “Help”, “Tools” or “Edit” facility).
+
+## Our Commitment to Data Security
+
+Konflux trains its administrators on our privacy policy guidelines and makes our privacy policy available to our partners. Our website uses Secure Socket Layer (SSL) technology, which encrypts your personal data when you send your personal information on our website. In addition, Konflux and its partners enter into confidentiality agreements which require that care and precautions be taken to prevent loss, misuse, or disclosure of your personal data.
+
+## Public Forums Reminder
+
+Konflux makes chat rooms and/or mailing lists available to its users. Please remember that any information that is disclosed in these areas becomes public information. Exercise caution when deciding to disclose your personal data. Although we value individual ideas and encourage free expression, Konflux reserves the right to take necessary action to preserve the integrity of these areas, such as removing any posting that is vulgar or inappropriate. It is in Konflux’s legitimate business interests to provide all users an accurate record of data and content provided in the public forums it maintains and uses; to maintain the integrity of that data and content for historical, scientific, and research purposes; and to provide an environment for the free exchange of ideas relevant and constructive to the development and propagation of open source software.
```

Comment:
Here at "vulgar or inappropriate" you could replace that with a link to the code of content - i.e. "removing any posting that is in breech of our [code of content](link goes here)".